pkg/deploy/api/v1/conversion_generated.go

```diff
@@ -196,9 +177,7 @@ func Convert_api_DeploymentCauseImageTrigger_To_v1_DeploymentCauseImageTrigger(i
 }
 
 func autoConvert_v1_DeploymentConfig_To_api_DeploymentConfig(in *DeploymentConfig, out *deploy_api.DeploymentConfig, s conversion.Scope) error {
-	if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {
-		defaulting.(func(*DeploymentConfig))(in)
-	}
+	SetDefaults_DeploymentConfig(in)
```

Comment:
We can defer this for after the rebase.  Custom conversions are still
correct, just inefficient.

On Thu, Jun 9, 2016 at 11:47 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/deploy/api/v1/conversion_generated.go
> <https://github.com/openshift/origin/pull/8856#discussion_r66467076>:
>
> > @@ -196,9 +177,7 @@ func Convert_api_DeploymentCauseImageTrigger_To_v1_DeploymentCauseImageTrigger(i
> >  }
> >
> >  func autoConvert_v1_DeploymentConfig_To_api_DeploymentConfig(in *DeploymentConfig, out *deploy_api.DeploymentConfig, s conversion.Scope) error {
> > -	if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {
> > -		defaulting.(func(*DeploymentConfig))(in)
> > -	}
> > +	SetDefaults_DeploymentConfig(in)
>
> You also need to update our custom conversions (conversions.go in the same
> directory). You actually need to go over all custom conversions in the repo
> and make sure we are defaulting correctly.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/8856/files/f956c25627008d75775d2042c6bf4ddbb8f63fce..96abf64e7e865e8cf944cb3f5444de10d4b89cbf#r66467076>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_pxGjoduPIKEEfufZlASGnYf8JKuSks5qKDWJgaJpZM4Ic8eO>
> .
>